python/ray/tests/BUILD

```diff
@@ -161,6 +160,7 @@ py_test_module_list(
   files = [
     "test_actor.py",
     "test_memstat.py",
+    "test_gcs_fault_tolerance.py",
```

Comment:
Can you move this to the Mac only flaky module list down below? I think you'd have to create a new py_test_module_list and tag it as `flaky-mac`

Reply:
If we do that, does it still run with Linux?

Reply:
Yes it should still with run on Linux on the regular build, but with Mac on the flaky build.